ppx/dune

```diff
@@ -0,0 +1,14 @@
+(library
+ (name reason_react_ppx)
+ (modules reason_react_ppx)
+ (public_name reason-react-ppx)
+ (flags :standard -w -9)
+ (kind ppx_rewriter)
+ (libraries ppxlib))
+
+(executable
+ (name standalone)
+ (modules standalone)
+ (package reason-react-ppx)
+ (public_name reason-react-ppx.standalone)
```

Comment:
Is there any reason to make this public? I think the ppx will always be consumed as a library.

Reply:
It can be useful to run the ppx transformation as an executable. I'm in favor of this one.

I'd name it to a better name though. I think it can even be called `reason-react-ppx`. it'll be installed in `bin/reason-react-ppx`.

Reply:
I didn't know was possible the same name as the lib